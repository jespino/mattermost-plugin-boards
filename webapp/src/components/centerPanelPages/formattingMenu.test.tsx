// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import '@testing-library/jest-dom'
import {act, render} from '@testing-library/react'

import React from 'react'

import {wrapIntl} from '../../testUtils'

import FormattingMenu from './formattingMenu'

describe('components/formattingMenu', () => {
    test('should match snapshot', async () => {
        let container
        await act(async () => {
            const result = render(wrapIntl(
                <FormattingMenu/>,
            ))
            container = result.container
        })
        expect(container).toMatchSnapshot()
    })
})
