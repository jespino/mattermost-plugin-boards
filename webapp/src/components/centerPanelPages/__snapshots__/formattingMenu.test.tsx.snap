// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/formattingMenu should match snapshot 1`] = `
<div>
  <div
    class="FormattingMenu"
  >
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-header-1"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-header-2"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-header-3"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-header-4"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-header-5"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-header-6"
      />
    </button>
    <span
      class="divider"
    />
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-bold"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-italic"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-strikethrough-variant"
      />
    </button>
    <span
      class="divider"
    />
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-link-variant"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-code-tags"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-code-block"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-quote-open"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-list-bulleted"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-format-list-numbered"
      />
    </button>
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-table-plus"
      />
    </button>
    <span
      class="divider"
    />
    <button
      class="IconButton size--small"
      type="button"
    >
      <i
        class="CompassIcon icon-plus"
      />
    </button>
  </div>
</div>
`;
